APIChangeReport.md

```diff
@@ -0,0 +1,183 @@
+
+  
+<!--
+ Copyright © 2019 and later: Unicode, Inc. and others.
+ License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# ICU4C API Comparison: ICU 70 with ICU 71 (m1)
```

Comment:
FYI it does say "(m1)" rather than just 71...

Reply:
Thx. Fixed now.